WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -468,6 +468,9 @@ public Parcelable saveState() {
             Bundle bundle = (Bundle) super.saveState();
             // This is a possible solution to https://github.com/wordpress-mobile/WordPress-Android/issues/5456
             // See https://issuetracker.google.com/issues/37103380#comment77 for more details
+            if (bundle == null) {
+                bundle = new Bundle();
+            }
             bundle.putParcelableArray("states", null);
```

Comment:
The comment above was intended for this line, but now it looks like it's about the null check. We should move the null check above the comment.